src/sentry/api/serializers/models/group.py

```diff
@@ -118,6 +118,9 @@ def serialize(self, obj, attrs, user):
         metadata = obj.data.get('metadata') or {
             'title': obj.message_short,
         }
+        # TODO(dcramer): remove in 8.6+
+        if event_type == 'error' and 'value' in metadata:
+            metadata['value'] = unicode(metadata['value'])
```

Comment:
Technically it was the `type` field that was busted:

![image](https://cloud.githubusercontent.com/assets/2153/15166193/68bbac00-16d1-11e6-9cf7-1639631cad4a.png)